android/app/build.gradle

```diff
@@ -0,0 +1,87 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlinx-serialization'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+    ndkVersion "21.0.6113669"
+
+    lintOptions {
+        disable 'NewApi'
+    }
+
+
+    defaultConfig {
+        applicationId "io.github.ja2stracciatella"
+        minSdkVersion 19
+        targetSdkVersion 26
+        versionCode 18
+        versionName "0.18.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        externalNativeBuild {
+            cmake {
+                 "-std=c++11"
+                arguments '-DBUILD_SDL_LIB=ON',
+                          '-DBUILD_LAUNCHER=OFF',
+                          '-DWITH_UNITTESTS=OFF',
+                          '-DWITH_RUST_BINARIES=OFF',
+                          '-DCMAKE_CXX_FLAGS=-fPIC',
+                          '-DCMAKE_C_FLAGS=-fPIC',
+                          '-DANDROID_APP_PLATFORM=android-16'
+            }
+        }
+
+        ndk {
+            // Specifies the ABI configurations of your native
+            // libraries Gradle should build and package with your APK.
+            // Currently we dont have armv7 as there is an issue with the SDL version we use
+            abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    externalNativeBuild {
+        cmake {
+            path "../../CMakeLists.txt"
+            version "3.10.2"
+        }
+    }
+    sourceSets {
+        main {
+            assets {
+                srcDirs = [ "../../assets" ]
+            }
+        }
+    }
+}
+
+dependencies {
```

Comment:
Can we use the pre-packaged SDL_android library from Maven? Or do we have some specific patches in the SDL source code?

Reply:
I think SDL_android on maven is a different SDL (SmartDeviceLink). And yes we apply a single patch to SDL.